src/pages/components/tabs/usage.mdx

```diff
@@ -451,6 +451,75 @@ secondary labels with line tabs or auto-width contained tabs.
 </Column>
 </Row>
 
+### Dismissible tabs
+
+Dismissible tabs allow users to close tabs providing a focused and relevant
+experience. Users can add or remove tabs as needed, accommodating future content
+additions or modifications without drastically changing the overall layout or
+structure.
+
+<Row>
+<Column colLg={8}>
+
+![Dismissible tabs](images/tabs-usage-15.png)
+
+</Column>
+</Row>
+
+<Caption>
+  Example of dismissible line tabs (top) and dismissible contained tabs (bottom)
+</Caption>
+
+#### When to use
+
+- Use dismissible tabs to offer flexibility and scalability in complex
+  interfaces that require users to create multiple sections or modules.
+- Use dismissible tabs for content created or curated by the user.
+- Use to focus a specific data set or search results.
+
+#### When not to use
+
+- Do not use when tabs contain frequently used or critical information.
+- Do not use as navigation.
+
+#### Dismissible tabs with icons
+
+Carbon provides an option to include icons within the dismissible tabs. Only use
+icons within dismissible tabs if all tabs will include icons. Do not mix
+dismisible tabs without icons with dismissible tabs with icons.
+
+#### Dismissing a tab
+
+When dismissing a tab, an inline warning or modal may be used when information
+contained in a tab will no longer be accessible or difficult to retrieve. Since
+warnings, especially modals, are highly disruptive, only use when the dismissal
+causes errors, unintentional deletions, or unsaved changes.
+
+#### Triggering a new tab
+
+There are various ways to trigger a new tab. The trigger button can visually
+change its shape and size depending on the use case. Keep the trigger close 
+enough to the new tab to associate the add action with the new tab item. The
+order of tabs can be ascending or descending depending on use case but do keep
+them in a sequential, logical order.
+
+If all tabs are dismissible, make sure a user understands how to trigger new
+tabs once the tabs are gone. Provide visual cues, such as a container or
+placeholder tab, so the user clearly understands that the trigger button is
+creating a new tab in a place they expect.
+
+<Row>
+<Column colLg={8}>
+
+![Dismissible tabs](images/tabs-usage-16.png)
```

Comment:
I don't think this proposed scenario is achievable with what is in code right now. 
<img width="801" alt="image" src="https://github.com/carbon-design-system/carbon-website/assets/40550942/dd822725-5fec-4b52-b531-215f9c7ecfd2">

We're enforcing that all tabs have to be dismissible within a `TabList` and we're setting the close icon ourselves. is there a requirement for "operable" icons outside of dismissible tabs? if so we need to open an issue for this

Reply:
yeah. with these docs i'm trying to tow the line between the coded component and a pattern. users will need some guidance around triggering a tab even though we don't provide code for it. i should make that a bit clearer though that it is not provided in the actual component.

Reply:
updated the image using a normal button instead of a tab styled button!